app/src/main/java/cm/aptoide/pt/app/view/donations/data/BaseResponse.java

```diff
@@ -0,0 +1,412 @@
+package cm.aptoide.pt.app.view.donations.data;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class BaseResponse {
```

Comment:
Please make this package private so we don't expose this to other features. This should be something we handle in the future.